### PR TITLE
E-1050 feat: add new font

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

**Comment by ifiri** [comment]
@dudiq ready for review

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "@endpass/class": "^0.20.1",
```
Comment (dudiq) [inline package.json:44]: and where new fonts declared? can't see any declaration for default texts.

Comment (ifiri) [reply package.json:44]: There used two variables: $headings-font-family (which was changed), using for headings, and `--endpass-ui-font-family-primary`, which inherited from `@endpass/ui`. So no another places for font-family here. When you will approve, you can check real font-family of text via devtools.

**Comment by ifiri** [comment]
@dudiq ready for approve

File: src/css/fonts.scss [hunk 2]
```diff
@@ -17,3 +17,45 @@
   font-style: normal;
   font-weight: 700;
 }
+
+@font-face {
```
Comment (lamartire) [inline src/css/fonts.scss:21]: image
Do not applied correctly to the all elements  in the application. Check it with any font inspector

Comment (ifiri) [reply src/css/fonts.scss:21]: As I know I should change only text fonts here

@kazurus

Comment (kazurus) [reply src/css/fonts.scss:21]: As I remember, we have another font for code. But all other text should be with new font. @dudiq could you remind me about code font?

Comment (dudiq) [reply src/css/fonts.scss:21]: @kazurus  `JetBrains Mono`

Comment (dudiq) [reply src/css/fonts.scss:21]: and `JetBrains Mono` font should used for `<code>` `<pre>` and `.code`

Comment (ifiri) [reply src/css/fonts.scss:21]: code font should stay unchanged

**Comment by ifiri** [comment]
@lamartire check again please, after investigations finally found the reason. Now all should looks fine.

**Pull request merged**